Kconfig.zephyr

```diff
@@ -82,6 +82,15 @@ config CODE_DATA_RELOCATION
 	  files should be specified in the CMakeList.txt file with
 	  a cmake API zephyr_code_relocation().
 
+config RELOCATION_SRAM_TEXT_ALIGN_SIZE
+	int "code relocation sram text section align size"
+	default 80000
```

Comment:
What does this mean? We are by default reserving 80 Kilobytes of space? Why so much, given that the offset in the binary was only 8 bytes? The help text needs to clearly explain the semantics on the value we set here.